includes/class-sensei-frontend.php

```diff
@@ -969,34 +966,6 @@ public function sensei_lesson_quiz_meta( ) {
 
 	} // End sensei_lesson_quiz_meta()
 
-	public function sensei_course_archive_meta() {
```

Comment:
This might not be called directly but it appears it is hooked in an action above (also removed)